### ISSUE TITLE
API/JSON incosistencies

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/deck#771]
Hi @juliushaertl 
This is rather a question/discussion, can really say if it is to be considered a feature, a bug orr to be ignored.

Please see below the actual (0.5.1) JSON element for a card retrieved in the Android client, see https://github.com/stefan-niedermann/nextcloud-deck/pull/16#issuecomment-445237500
```
{
"title":"Fix some naughty bug",
"description":"getRandom always returns four\nThis is line two\nThis is line three",
"stackId":8,
"type":"plain",
"lastModified":1544169560,
"lastEditor":"andy",
"createdAt":1544169433,
"labels":[
{"title":"To review","color":"317CCC","boardId":"5","cardId":null,"id":18},
{"title":"Later","color":"F1DB50","boardId":"5","cardId":null,"id":20}
],
"assignedUsers":[
{"id":2,"participant":{"primaryKey":"andy","uid":"andy","displayname":"Andy"},"cardId":"24"}
],
"attachments":null,
"attachmentCount":0,
"owner":"andy",
"order":999,
"archived":false,
"duedate":"2018-12-14T07:59:20+00:00",
"deletedAt":0,
"commentsUnread":0,
"id":24,
"overdue":0
}
```
* the label/```cardId``` of the labels is always `null` and  #774 
* the label/`boardId` is a `String` instead of a `long` #774 
* the assignedUser/```cardId``` of the assignedUsers is is a `String` instead of a `long` #774 
* `lastModified`, `createdAt` and `duedate` are all dates/timestamps while being mixed types long/date-string where the date string ships the timezone (kudos for that!) #809 

Any feedback is highly appreciated, as always :) ❤️

**Comment by juliushaertl** [comment]
Thanks for reporting. This is actually a bug. I have a fix ready for the first 3 list items (#774), the date type unification however need a deeper look, since changing the types will probably also need some adjustments at the frontend of deck. I'm looking into this.

**Comment by AndyScherzinger** [comment]
Thanks a lot for your help and efforts @juliushaertl! Highly appreciate your work!
